pkg/sql/colexec/execgen/testdata/type_template

```diff
@@ -0,0 +1,76 @@
+template
+package main
+
+
+func main() {
+  a_int(3, 4)
```

Comment:
I think that doing something like
```
// execgen:call<int>
a(3, 4)
```
wouldn't be that hard, and allows the code generator to pick the right implementation